src/components/Header/Header.css

```diff
@@ -23,7 +23,7 @@ h1 {
   width: 2rem;
 }
 
-.home-icon:hover {
```

Comment:
We'll be adding this to a media-query. Not sure how that will fit in folder/file structure wise, though. I'll make an issue for it.